client/layout/guided-tours/docs/API.md

```diff
@@ -260,9 +260,11 @@ Some examples:
 // detect state (has-thumbnail) and select a child (featured image)
 <Step target=".reader-post-card.has-thumbnail .reader-post-card__featured-image">
 
-// target id (notice the required space before #)
-<Step target=" #header">
+// target id
+<Step target="#header">
 
-// target element (again, space at the beginning)
+// target element
 <Step target=" body">
```

Comment:
woah! only seeing this now. probably note here that this is kind of a hack that you hopefully wouldn't ever need to use ...?